critiquebrainz/db/users.py

```diff
@@ -0,0 +1,35 @@
+from hashlib import md5
+import sqlalchemy
+from critiquebrainz import db
+
+
+def avatar(item):
+    url = "https://gravatar.com/avatar/{0}{1}"
+    email = item['email']
+    if email and item['show_gravatar']:
+        return url.format(md5(email.encode('utf-8')).hexdigest(), "?")
+    else:
+        return url
```

Comment:
@tigranl , 
gravatars generates images based on md5 hashes provided to it. http://en.gravatar.com/site/implement/images/

And we required a fix where people who don't provide email ids/ don't allow their gravatar to be shown should also get an image displayed. As their IDs were unique, we used it to generate the new hashes.
https://github.com/metabrainz/critiquebrainz/pull/67#discussion_r91604203